03_instructional_team/markdown_slides/slides_01.md

```diff
@@ -39,34 +39,36 @@ $ echo "Data Sciences Institute"
 
 ---
 
-# About Us (Amanda)
+# About Us (Ernani)
 
-- UofT Statistics & Math undergrad (4th year)
-- Considering grad school in Stat/Biostat
-- Self-taught SQL through online resources
-- Have experiences in both research and industry 
-- Hobbies: piano, puzzle, Netflix
+- UofT (DSI) & Master of Engineering in S&T at McMaster University
+- Electrical & Telecom Engineer
+- DSI cohort 3 alumni
+- Cloud AWS Architecture, Unix / Linux & Data Science
+- Facial Action Coding System Instructor (Paul Ekman / F-M Group International)
+- Hobbies: Photography, Math challenges, Reading, Neuroscience
 
-![bg right:35% w:350](./images/01_amanda.png)
+![bg right:35% w:350](./images/01_ernani.png)
 
 ---
 
-# About Us (Sidra)
+# About Us (James)
 
-- Bachelor’s in Biomedical Engineering from NED-UET
-- Postgraduate diploma in data analytics from  Toronto School Of Management.
-- Machine Learning Software Foundations Certificate from DSI-University Of Toronto.
-- 4+ years of experience in business intelligence and data analysis.
-- Engaged in  Continuously Learning and Upskilling my skills
+- UofT Engineering undergrad and master 🦫
+- Licensed engineer in an oil & gas company sending robots into pipelines 🤘
+- DSI cohort 3 alumni
+- Self-taught woodworking hobbyist 🪚
+- Real hobbies: motorcycle 🛵, leisure skating ⛸️, camping ⛺️
 
-![bg right:35% w:350](./images/01_sidra.png)
+![bg right:35% w:350](./images/01_james.png)
 
 ---
 
 # About Us (Niyaz)
 
 - Master's in Artificial Intelligence with a deep understanding of machine learning algorithms, data science, and advanced computational techniques
 - 15 years of experience in software development, database design, and query optimization
+- DSI cohort 2 alumni
```

Comment:
Can you say "future proofed participant"? We stay away from terminology like alumni.

Reply:
ha lets just do "DSI cohort 2"

Reply:
done